### ISSUE TITLE
"Toast" is not announced by Voiceover or Talkback

### ISSUE THREAD

**Issue by simonflk** [GeekyAnts/NativeBase#4985]
### Description

I expect the contents of the alert to be read by the screenreader when it appears on screen.

### CodeSandbox/Snack link

https://snack.expo.dev/@simonflk/nativebase-alert-example

### Steps to reproduce

1. Enable Voiceover on iOS or Talkback on Android device
2. Open snack link
3. Press any button to trigger a toast
4. Wait for screenreader to read contents of toast

### NativeBase Version

3.4.3

### Platform

- [X] Android
- [ ] CRA
- [ ] Expo
- [X] iOS
- [ ] Next

### Additional Information

iOS 15.4.1
Android 12 (Pixel 5, April 2022 update)

Docs says:
* On Android and Web, Toast renders under a View with accessibilityLiveRegion which announces the content rendered inside it to screen reader devices.
* On iOS, accessibilityLiveRegion is not supported yet, so we use the 
[accessibilityAnnouncement](https://reactnative.dev/docs/accessibilityinfo#announceforaccessibility)
 to announce the content.

But this is not happening for me

**Comment by Viraj-10** [comment]
Hi @simonflk, Thanks for reporting the issue. We will look into it.
